app/utils/Strings.scala

```diff
@@ -38,6 +38,25 @@ object Strings {
     Random.alphanumeric.filter(_.isDigit).take(length).mkString
   }
 
+  def decimal(length: Int): String =
+    Seq.fill(length)(Gen.choose(1, 9).sample.getOrElse(1)).mkString
+
+  def numeric8(): String = {
+    val first  = Gen.choose(1, 2).sample.getOrElse(1)
+    val second = Gen.choose(0, 9).sample.getOrElse(9)
+
+    Seq(
+      first.toString,
+      second.toString,
+      first.toString,
+      second.toString,
+      "0",
+      Gen.choose(1, 9).sample.getOrElse(5).toString,
+      "0",
+      Gen.choose(1, 9).sample.getOrElse(7).toString,
```

Comment:
If you add a def to run Gen.choose(1, 9).sample.getOrElse(1).mkString you could reuse that a few times in this file. Just as it is a pretty common use case atm.